utils/fields.py

```diff
@@ -86,6 +86,63 @@ def _validate_vector_tile_url(value: typing.Any):
 PydanticUlid = typing.Annotated[str, Field(strict=True, pattern=r"^[0-9A-Z]{26}$")]
 
 
+PydanticIsPano = typing.Annotated[
```

Comment:
We do not need to create new types for each field.
If it's just for description we need to find other ways to extend that.